commands/orchestrationcommands/stackresourcecommands/commands.go

```diff
@@ -0,0 +1,17 @@
+package stackresourcecommands
+
+import "github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+
+var commandPrefix = "orchestration stack-resource"
```

Comment:
There is only one "kind" of resource, a stack resource. A stack is made of resources of different types (e.g. `OS::Nova::Server`).